configs/sgv e-bike study.nrel-op.json

```diff
@@ -0,0 +1,95 @@
+{
+  "url_abbreviation": "sgv e-bike study",
+  "version": 1,
+  "ts": 1741106543,
+  "server": {
+    "connectUrl": "https://sgv e-bike study-openpath.nrel.gov/api/",
+    "aggregate_call_auth": "user_only"
+  },
+  "opcode": {
+    "autogen": false,
+    "subgroups": [
+      "default",
+      "test"
+    ]
+  },
+  "intro": {
+    "program_or_study": "program",
+    "start_month": "03",
+    "start_year": "2025",
+    "mode_studied": "",
+    "program_admin_contact": "Aaron Fellhoelter (email: aaronf1@vols.utk.edu)",
+    "deployment_partner_name": "University of Tennessee, Knoxville and Portland State University",
+    "translated_text": {
+      "en": {
+        "deployment_partner_name": "University of Tennessee, Knoxville and Portland State University",
+        "deployment_name": "San Gabriel Valley E-bike Study",
+        "summary_line_1": "The San Gabriel Valley Council of Governments (SGVCOG) and Active San Gabriel Valley (ASGV) are developing and administering multiple e-bike strategies. The program focuses on removing cost barriers to e-bike use.",
+        "summary_line_2": "If you are a program Participant, this app can help you log your trips and allow the San Gabriel Valley Council of Goverments to understand travel patterns.",
+        "summary_line_3": "",
+        "short_textual_description": "The San Gabriel Valley Council of Governments and Active San Gabriel Valley (ASGV) are developing policy for e-bike bikeshares and incentives to promote more sustainable and equitable forms of travel. This OpenPath application helps us to understand the travel patterns of participants.",
+        "why_we_collect": "As a participant of the study, the OpenPath app will collect data on your travel modes, distance, and frequency. This data will help us to gain a better understanding of the travel characteristics of participants, and how we can improve the program to make the San Gabriel Valley a more livable and healthier place for its citizens.",
+        "research_questions": []
```

Comment:
I would suggest that you fill out at least one research question, otherwise the various templates will look ugly.